scripts/jenkins/ardana/manual/requirements.txt

```diff
@@ -0,0 +1,38 @@
+ansible==2.7.7
+appdirs==1.4.3
+asn1crypto==0.24.0
+bcrypt==3.1.6
+certifi==2018.11.29
+cffi==1.12.1
+chardet==3.0.4
+cryptography==2.5
```

Comment:
do we really need to have "frozen" dependencies here? I think we should have more liberal definitions.. why does it need to be locked to a specific version (same for all the other lines..)

Reply:
I have no specific reason to have those frozen dependencies. Simple got it from the virtual environment.

Following your suggestion, I will make a change to improve that.